src/main/target/common.h

```diff
@@ -40,6 +40,7 @@
 #define TELEMETRY
 #define TELEMETRY_LTM
 #define TELEMETRY_FRSKY
+#define USE_RCSPLIT
```

Comment:
Please move this to `FLASH_SIZE > 128` section. It won't fit into F1-based targets.